### ISSUE TITLE
Payday2 won't open with mods on

### ISSUE THREAD

**Issue by WaahAgain** [Kamikaze94/WolfHUD#834]
# Prerequisites

This is my first time ever posting on GitHub so Idk if I'm doing this right.

# Expected Behavior

I want to open Payday 2.

# Current Behavior

I can't open Payday 2.

# Failure Information (for bugs)
#### Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.

So, after the new update, I realized that WolfHUD was gone. I got out of the game and reinstalled WolfHUD, I opened the game and the mods weren't there. So I deleted the mods, restarted steam, and tried again. I reinstalled it by extracting and whatnot, and now Payday 2 won't even open. I've tried validating the game files a couple of times, don't remember when.

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Press play
2. It says running
3. It stops saying running and nothing happens

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Operating System: Windows 10
* BLT version: I think I have a "SuperBLT" I got it from here, https://superblt.znix.xyz/
* Other mods installed: None
* ...

## BLT log

Please include the contents of your BLT log around the time the bug appeared.  
You can find the BLT log inside your games "/mods/logs/" folder.

My BLT log's latest one was before the update, so I actually don't have anything to paste here. The log for today 5/3/2020 is empty.
[2020_04_28_log.txt](https://github.com/Kamikaze94/WolfHUD/files/4572196/2020_04_28_log.txt)
So, here is my latest one, maybe that helps.

## Crash log(s)

If the bug is causing a crash, please provide the contents of your crash log.  
You can find your crash log in "%localappdata%/Payday 2/crash.txt"

  Application has crashed: C++ exception
[string "lib/units/props/unodevicebase.lua"]:71: attempt to index a nil value

SCRIPT STACK

_function() core/lib/managers/coresequencemanager.lua:4545
activate_callback() core/lib/managers/coresequencemanager.lua:4538
activate() core/lib/managers/coresequencemanager.lua:2017
activate_callback() core/lib/managers/coresequencemanager.lua:2814
activate() core/lib/managers/coresequencemanager.lua:2017
run_sequence() core/lib/managers/coresequencemanager.lua:2478
run_sequence() core/lib/managers/coresequencemanager.lua:380
update_start_time_callbacks() core/lib/managers/coresequencemanager.lua:526
update() core/lib/managers/coresequencemanager.lua:479
core/lib/setups/coresetup.lua:494

-------------------------------

Callstack:

         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     zip_get_name                                        
         payday2_win32_release  (???)     zip_get_name                                        

-------------------------------

Current thread: Main

-------------------------------

System information:
	Application version : 1.93.850.
	CPU : Intel(R) Core(TM) i5-7300HQ CPU @ 2.50GHz (2 cores); SSE; SSE2; SSE3; SSSE3; SSE4.1; SSE4.2
	DirectX : 12.0 
	GPU : NVIDIA GeForce GTX 1050 Ti / nvldumd.dll[25.21.14.2545]
	Language : english
	Memory :     8058MB 468KB
	OS : 6.2.9200 () 0x300-0x1 (64 bits)
	Physics : threaded
	Renderer : DX9 threaded
	Sound : Realtek Semiconductor Corp. (Speakers / Headphones (Realtek Audio))

Please, if I'm missing something, I apologize, this is my first time.

**Comment by MEXAHOTABOP** [comment]
Hello
1) delete payday 2/mods/base
2) delete `IPHLPAPI.dll` and `WSOCK32.dll` (if exist) from game directory
3) download https://znix.xyz/random/payday-2/SuperBLT/latest.php and place in game directory

**Comment by WaahAgain** [comment]
> Hello
> 
> 1. delete payday 2/`mods/base`
> 2. delete `IPHLPAPI.dll` and `WSOCK32.dll` (if exist) from game directory
> 3. download https://znix.xyz/random/payday-2/SuperBLT/latest.php and place in game directory

So, I did this, I can open the game now, yay. But my WolfHUD still isn't there.

**Comment by MEXAHOTABOP** [comment]
Then fresh logs needed
I'll look at them tomorow

**Comment by WaahAgain** [comment]
> ne

It won't make new logs. So...

**Comment by MEXAHOTABOP** [comment]
you had superblt settings in the game?

**Comment by WaahAgain** [comment]
No, when I open the game, where the mod setting things is supposed to be, it isn't there.

> Do you have a superblt settings in the game?

**Comment by MEXAHOTABOP** [comment]
Then, i guess you need to ask in blt issue tracker https://gitlab.com/znixian/payday2-superblt/-/issues

**Comment by WaahAgain** [comment]
Okay so, after reading on a few of the issues, somebody had something similar to mine, and said something about "IPHLAP" or something like that and a "WSOCK" and said to delete the "IPHLAP" so I did, and now it's fixed. Thank you so much for your help! :)

> Then, i guess you need to ask in blt issue tracker https://gitlab.com/znixian/payday2-superblt/-/issues
